fs/log.go

```diff
@@ -77,25 +77,38 @@ var LogPrint = func(level LogLevel, text string) {
 // LogPrintf produces a log string from the arguments passed in
 func LogPrintf(level LogLevel, o interface{}, text string, args ...interface{}) {
 	out := fmt.Sprintf(text, args...)
-	if o != nil {
-		out = fmt.Sprintf("%v: %s", o, out)
-	}
+
 	if Config.UseJSONLog {
+		fields := logrus.Fields{}
+		switch o.(type) {
```

Comment:
This needs to do something for case string: doesn't it?  Maybe you don't need the type switch at all and can just do the same thing for all types?

Reply:
Yes, it does. I thought that was to prevent redundant information in logs but it's fine to apply the format for all types. I'll update it.